content/applications/hr/payroll.rst

```diff
@@ -59,25 +59,21 @@ are correctly entered.
 New work entry type
 ~~~~~~~~~~~~~~~~~~~
 
-To create a new work entry type, click the :guilabel:`Create` button. Enter the information on the
+To create a new work entry type, click the :guilabel:`New` button, and enter the information on the
```

Comment:
I think Work Entry Type should have a guilabel, or at least be capitalized/italicized when it is first introduced. (It appears to have a guilabel in some of the other Payroll docs, but that is specific to a field, so I realize there is some gray area)

Reply:
I honestly don't recall the logic on why we didn't here- I think it was because it was being used as a 'generic' phrase but I totally agree with you .